### DISCUSSION TITLE
Moving 3.4 out of the pre-release phase?

### DISCUSSION THREAD

**Discussion by bilderbuchi** [krvajal/vscode-fortran-support#1115] [Q&A - deprecated see Fortran Discourse] [answered]
Hi! Is it intentional that the 3.4 version is still marked as a "pre-release"? When I try to add this extension to VS Code, I'm only being offered 3.2.0 (from July 2022!), with a "pre-release version" available (`v3.4.2024061901`). 
Possibly this is because https://github.com/fortran-lang/vscode-fortran-support/releases contains 3.4.0 only as a pre-release, but I'm not sure at all how this works with respect to the VS Code Marketplace?

Is it planned to switch the 3.4 versions to a "full" release cadence at some point, as you seem to push out a steady stream of releases/features ❤️, so that seems like a non-problematic change to me? 
I'm asking because I'm working out of a devcontainer that is configured to install this extension, and that only sees 3.2.0 -- I'd have to find a way if one can make pre-releases available for a devcontainer setup, somehow.

**Comment by gnikit** [comment, marked as answer]
> Is it planned to switch the 3.4 versions to a "full" release cadence at some point, as you seem to push out a steady stream of releases/features ❤️, so that seems like a non-problematic change to me?

Yes, there absolutely is a plan. Feature-wise the pre-release and the v4 release will be almost identical. The reasons why I have not released it yet are related to our Organisation setup. I need to do a bunch of admin stuff. I hope to get them done over by September.

**Reply by bilderbuchi** [reply]
Thanks, I understand! 

In the meantime, should anybody need that, you can get a prerelease version in a devcontainer by specifying it [like this](https://github.com/microsoft/vscode/issues/149383#issuecomment-1132192731): `"fortran-lang.linter-gfortran@prerelease"`
